types/topology.go

```diff
@@ -416,7 +416,7 @@ func (t *Topology) GetSysCtl(name string) map[string]string {
 	return nil
 }
 
-// Return the Subject Alternative Name configuration for the given node.
+// GetSANs Return the Subject Alternative Name configuration for the given node.
```

Comment:
thanks for fixing the comments patter
could you also decapitalise the next word after the method name?

Reply:
done!